libafl/src/executors/command.rs

```diff
@@ -158,16 +172,116 @@ where
     }
 }
 
-/// A `CommandExecutor` is a wrapper around [`std::process::Command`] to execute a target as a child process.
+/// Linux specific [`CommandConfigurator`] that leverages `Ptrace`
+///
+/// This configurator was primarly developed to be used in conjunction with
+/// [`crate::executors::hooks::intel_pt::IntelPTHook`]
+#[cfg(all(feature = "std", target_os = "linux"))]
+#[derive(Debug, Clone, PartialEq, Eq, TypedBuilder)]
+pub struct PtraceCommandConfigurator {
```

Comment:
Assuming this would be process trace, maybe `PTrace` with second letter uppercase? Like `CString`. 100% up to you though